packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -17,26 +17,33 @@ class DiffPaneController {
 
   final SnapshotTaker snapshotTaker;
 
+  final _snapshots = ListValueNotifier(<DiffListItem>[InformationListItem()]);
+
   /// The list contains one item that show information and all others
   /// are snapshots.
-  final snapshots = ListValueNotifier(<DiffListItem>[InformationListItem()]);
+  ValueListenable<List<DiffListItem>> get snapshots => _snapshots;
+
+  final _selectedIndex = ValueNotifier<int>(0);
+  ValueListenable<int> get selectedIndex => _selectedIndex;
```

Comment:
always order getter before private field. Order should be

getter (if present)
field
setter (if present)